docs/process-for-changing-ion-schema.md

```diff
@@ -0,0 +1,173 @@
+We believe that it should be easy to iterate on Ion Schema, and so the most critical feedback that we can receive is whether something we are introducing is a misfeature or could otherwise cause problems later on. Feedback about the omission of desired features (or part of a feature) will also be considered, but we may decide to defer solving it if there are no backwards compatibility issues that the proposed feature would introduce.
+
+After feedback has been gathered from Ion Schema users and subject-matter experts, the Ion Schema **maintainers** will incorporate the feedback and make the final decision on the RFC.
+
+The outcome of this process will be shared on the `ion-schema` GitHub issue, resulting in the PR being merged or closed as appropriate.
+
+## Creating an RFC for a new Ion Schema version
+
+When we should create a new version depends on our subjective judgment of whether it is a good time for a new version. However, we should consider:
+
+* Are we still in the midst of implementing the last Ion Schema version?
+* How many accepted features are waiting to be released in a new version of the ISL spec?
+* Has anyone specifically asked for a new version to be released?
+* Do we have the bandwidth to implement the new version in Ion Schema libraries?
+* Has a **contributor** outside the team of **maintainers** offered to help implement the new version?
+
+We should be transparent if we do not have the bandwidth to implement a new version, but we should not stand in the way of someone who is willing to invest their own time in improving Ion Schema.
+
+# FAQ
+
+#### **Why is there no private comment period?**
+
+Ion Schema is an open source project, so we are going to try to do our design work out in the open too. We want interested parties to be able to share their thoughts and have a say in the direction of Ion Schema. (See [opensource.guide – Keep Communication Public](https://opensource.guide/best-practices/#keep-communication-public).)
+
+#### What if I have feedback that contains confidential information?
+
+If you are employed by Amazon, refer to internal policies about sharing confidential information.
+
+If you are not employed by Amazon, please do not share any confidential information on GitHub or directly with the **maintainers**. If you need additional time to rework your feedback so that it does not include confidential information, you may ask for the comment period to be extended.
+
+Regardless of the source, the **maintainers** will attempt to publicly summarize all private feedback.
+
+#### Can a feature RFC become “un-accepted”?
+
+Yes. An RFC can be “un-accepted” by another RFC that explicitly revokes or supersedes a prior RFC. This “un-acceptance” RFC is expected to go through the same process as any other RFC.
```

Comment:
```suggestion
Yes. An RFC can be “un-accepted” by another RFC that explicitly revokes or supersedes a prior RFC. This “un-accepted” RFC is expected to go through the same process as any other RFC.
```

Reply:
"Unacceptance RFC" is intended to mean "the RFC that proposes revoking a previously accepted RFC", so changing it to "unacctepted RFC" would incorrectly change the meaning here. However, the fact that you thought to suggest this change indicates that my language was not clear enough on this point. I'll try to think of a better way to phrase it, and I'm open to suggestions if you have any.

Reply:
We can say "reverted" instead of "un-accepted", and we can say "An RFC that reverts or supersedes a previous RFC is expected to go through the same process as any other RFC."

Reply:
```suggestion
Yes. An RFC can be reverted by another RFC that explicitly revokes or supersedes a prior RFC. An RFC that reverts or supersedes a previous RFC is expected to go through the same process as any other RFC.
```